engine/src/main/scala/cromwell/engine/db/slick/ExecutionInfoComponent.scala

```diff
@@ -45,6 +47,16 @@ trait ExecutionInfoComponent {
     } yield (execution, executionInfo)
   )
 
+  val runningExecutionsAndExecutionInfosByWorkflowId = Compiled(
+    (id: Rep[String]) => for {
+      executionInfo <- executionInfos
+      execution <- executionInfo.execution
+      workflowExecution <- execution.workflowExecution
+      if workflowExecution.workflowExecutionUuid === id
```

Comment:
Should check for workflow execution in status running.  Sometimes ops will set a workflow to a terminal state if they want it ignored.